### PR TITLE
Update tested linux versions to match latest agents

### PR DESCRIPTION
**Pull request by matt-richardson**
Brings the docs up to date with the linux versions we test currently against

### CODE REVIEW

File: docs/infrastructure/deployment-targets/linux/index.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ hideInThisSectionHeader: true
 Linux servers can be configured as [deployment targets](/docs/infrastructure/deployment-targets/index.md) in Octopus.   
 
 The Octopus Server can communicate with Linux targets via SSH or the [Linux Tentacle](https://octopus.com/docs/infrastructure/deployment-targets/linux/tentacle).  
+The Octopus Server can communicate with Linux targets via SSH or the [Linux Tentacle](https://octopus.com/docs/infrastructure/deployment-targets/linux/tentacle).
```
Comment (michaelnoonan) [inline docs/infrastructure/deployment-targets/linux/index.md:11]: Copy-paste mistake? This line seems duplicated.

File: docs/infrastructure/deployment-targets/linux/index.md [hunk 2]
```diff
@@ -20,14 +21,14 @@ The following platforms are explicitly supported (we run automated tests against
 - Ubuntu 18.04 LTS
 - Ubuntu 16.04 LTS
 - Redhat (RHEL) 7.2
-- Centos 7.6
+- Centos 7.7
```
Comment (michaelnoonan) [inline docs/infrastructure/deployment-targets/linux/index.md:24]: I've checked all those versions against the buildagentautomation repository. They look correct.

File: docs/infrastructure/deployment-targets/linux/index.md (hunk 1 above)
Comment (matt-richardson) [reply docs/infrastructure/deployment-targets/linux/index.md:11]: No idea how that happened

Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/linux/index.md:11]:
```suggestion
```

**Pull request merged**